src/lib_ccx/params.c

```diff
@@ -375,37 +375,37 @@ void print_usage(void)
 	mprint("Output file segmentation:\n");
-	mprint("    -outinterval x output in interval of x seconds\n");
+	mprint("   --outinterval x output in interval of x seconds\n");
 	mprint("   --segmentonkeyonly -key: When segmenting files, do it only after a I frame\n");
 	mprint("                            trying to behave like FFmpeg\n\n");
 	mprint("Network support:\n");
-	mprint("            -udp port: Read the input via UDP (listening in the specified port)\n");
+	mprint("           --udp port: Read the input via UDP (listening in the specified port)\n");
 	mprint("                       instead of reading a file.\n\n");
-	mprint("            -udp [host:]port: Read the input via UDP (listening in the specified\n");
+	mprint("           --udp [host:]port: Read the input via UDP (listening in the specified\n");
 	mprint("                              port) instead of reading a file. Host can be a\n");
 	mprint("                              hostname or IPv4 address. If host is not specified\n");
 	mprint("                              then listens on the local host.\n\n");
-	mprint("            -udp [src@host:]port: Read the input via UDP (listening in the specified\n");
+	mprint("           --udp [src@host:]port: Read the input via UDP (listening in the specified\n");
 	mprint("                              port) instead of reading a file. Host and src can be a\n");
 	mprint("                              hostname or IPv4 address. If host is not specified\n");
 	mprint("                              then listens on the local host.\n\n");
-	mprint("            -sendto host[:port]: Sends data in BIN format to the server\n");
+	mprint("           --sendto host[:port]: Sends data in BIN format to the server\n");
 	mprint("                                 according to the CCExtractor's protocol over\n");
 	mprint("                                 TCP. For IPv6 use [address]:port\n");
-	mprint("            -tcp port: Reads the input data in BIN format according to\n");
+	mprint("           --tcp port: Reads the input data in BIN format according to\n");
 	mprint("                        CCExtractor's protocol, listening specified port on the\n");
 	mprint("                        local host\n");
-	mprint("            -tcppassword password: Sets server password for new connections to\n");
+	mprint("           --tcppassword password: Sets server password for new connections to\n");
 	mprint("                                   tcp server\n");
-	mprint("            -tcpdesc description: Sends to the server short description about\n");
+	mprint("           --tcpdesc description: Sends to the server short description about\n");
 	mprint("                                  captions e.g. channel name or file name\n");
 	mprint("Options that affect what will be processed:\n");
-	mprint("          -1, -2, -12: Output Field 1 data, Field 2 data, or both\n");
+	mprint("         --1, --2, --12: Output Field 1 data, Field 2 data, or both\n");
```

Comment:
-- is used for long args and - for short. 
-1 is short.

Reply:
AFAIK, there is no short/long version of 1,2,12 though. I could handle this case manually in the GUI, but then don't want the code to get messy there :P 
Can I add something like --field1 --field2 just for GUI purpose?

Reply:
Some args have a short version, some a long version, and some both. The GUI needs to be deal with that - just adding "--" to every arg because it's convenient for you it's not a good reason to break scripts that rely in -1 and not --1.

Just have an array of the proper form for each arg and do a look up...

Reply:
`if (strcmp(argv[i], "-1") == 0 || strcmp(argv[i], "--1") == 0)` Won't break any scripts though